Adyen/Validators/BrazilSocialSecurityNumberValidator.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+@_spi(AdyenInternal)
+public final class BrazilSocialSecurityNumberValidator: CombinedValidator, StatusValidator {
+    
+    private enum Constants {
+        static let firstMaxLength = 11
+        static let secondMaxLength = 14
+    }
+    
+    public let firstValidator: Validator
```

Comment:
Maybe not for here but would be nice to be able to have a list of validators and an extension on `[Validator]` that allow use of || and && on the validators in a scalable way.